cirq/sim/clifford/clifford_simulator_test.py

```diff
@@ -291,6 +291,19 @@ def test_clifford_stabilizerStateChForm_repr():
     assert repr(state) == 'StabilizerStateChForm(num_qubits=2)'
 
 
+def test_clifforf_circuit_SHSYSHS():
```

Comment:
I suppose a simple test would check whether XZ and Y yield the same state up to global phase, but this should work just as well.

Reply:
I was looking for something that could be exposed as a measurement. The error in the Y gate can only be exposed when there is a Y stabilize, and this is not the case in the proposed test.

Reply:
Fair enough. Thank you for adding the test!